CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ list(APPEND CMAKE_MODULE_PATH
 include(pkg-utils)
 include(project-utils)
 
-
+option(snapshots "Build snapshots" OFF)
```

Comment:
I think this should explain a bit more what this actually does. For once as I see it it only affects building the installer. So that should be mentioned here. Furthermore it might quickly explain what changes are made to the installer if it is built for a snapshot :eyes:

Reply:
I will add more to the description.

Reply:
Updated.